src/frontends/tensorflow/tests/CMakeLists.txt

```diff
@@ -10,6 +10,15 @@ if (NOT ENABLE_INTEL_CPU)
     set(EXCLUDED_TESTS ${CMAKE_CURRENT_SOURCE_DIR}/compilation.cpp)
 endif()
 
+ov_check_pip_packages(REQUIREMENTS_FILE "${CMAKE_CURRENT_SOURCE_DIR}/requirements.txt"
+                      MESSAGE_MODE WARNING
+                      WARNING_MESSAGE "TensorFlow testing models weren't generated, some tests will fail due models not found"
+                      RESULT_VAR tensorflow_FOUND)
+
+if(tensorflow_FOUND)
+    set(ctest_labels LABELS OV UNIT)
+endif()
```

Comment:
Is it possible to miss that tests actually does not work if modules are not found? Can we force running the tests on concrete platforms where we expect tests executing?